pym/gentoolkit/eclean/search.py

```diff
@@ -561,6 +561,8 @@ def findPackages(
     # to support FEATURES=binpkg-multi-instance.
     dead_binpkgs = {}
 
```

Comment:
I'd loose the blank line here

Reply:
do you want me sending another PR to address this? or will you amend it while merge it?